htdocs/api/v0.0.3-dev/MedhubConsent.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+
+namespace Loris\API;
+use http\Env\Response;
+
+set_include_path(get_include_path() . ":" . __DIR__);
+require_once 'APIBase.php';
+
+
+
+class MedhubConsent extends APIBase {
+
+
+
+    var $requestData;
+
+
+    /**
+     *  Candidates request handler
+     *
+     * @param string $method The HTTP request method of the request
+     * @param array  $data   The data that was POSTed to the request
+     */
+
+
+
+    public function __construct($method, $data=null)
+    {
+
+
+
+
+        $this->AllowedMethods = [
+            'PUT'
+        ];
+
+        $this->RequestData    = $data;
+
+        parent::__construct($method);
+    }
+
+
+
+    public function handlePUT()
+    {
+
+        $token = null;
```

Comment:
no need to set it to null, you only set it below directly with the correct value. this line can be removed